modules/ROOT/pages/dw-core-functions-minusminus.adoc

```diff
@@ -41,22 +43,46 @@ output application/json
 }
 ----
 
+Note: The result is the same as 
+==== Source
 
+[source,DataWeave, linenums]
+----
+%dw 2.0
+output application/json
+---
+{ "a" : [0, 1, 1, 2] - 1 - 2 }
+----
+
+==== Output
+
+[source,JSON,linenums]
+----
+{
+  "a": [0]
+}
+----
 [[minusminus2]]
 == &#45;&#45;&#40;{ &#40;K&#41;?: V }, Object&#41;: { &#40;K&#41;?: V }
 
-Removes specified key-value pairs from an object.
+Removes key-value pairs from a source object (the first parameter) if the same key/value appears in the second parameter's object. 
 
 
 === Parameters
 
 [%header, cols="1,3"]
 |===
 | Name   | Description
-| `source` | The object.
-| `toRemove` | Objects to remove from the source object.
+| `source` | The source object whose key/values might be removed (of type `Object`).
+| `toRemove` | The key/value pairs to remove from the source object (of type `Object`). The key/values in this second object to need not repeat; every matching key/value is removed from the source object. 
```

Comment:
@ethanport here the same, " The key/values in this second object to need not repeat;" Did you refer _The key-values pair in the second object can not be repeated/duplicated_ ?